app/src/pages/inside/membersPage/modals/permissionMapModal/permissionMap/permissionMap.jsx

```diff
@@ -23,110 +17,102 @@ export class PermissionMap extends Component {
     intl: {},
   };
   generateTableCells = (permission) => {
-    const roles = [ADMINISTRATOR, PROJECT_MANAGER, MEMBER, OPERATOR, CUSTOMER];
-    const hasPermission = (
-      <td className={cx('col', 'has-permission')}>
-        <i className={cx('rp-icons', 'rp-icons-check')} />
-      </td>
+    const { intl } = this.props;
+    const roles = [PROJECT_MANAGER, MEMBER, OPERATOR, CUSTOMER];
+    const ownerPermission = (
+      <div title={intl.formatMessage(ROLE_NAMES.ownTitle)} className={cx('permission-container')}>
+        <div className={cx('owner-permission')} />
+      </div>
     );
-    const noPermission = <td className={cx('col')} />;
+    const notOwnerPermission = (
+      <div
+        title={intl.formatMessage(ROLE_NAMES.notOwnTitle)}
+        className={cx('permission-container')}
+      >
+        <div className={cx('not-owner-permission')} />
+      </div>
+    );
+
     return roles.map((role) => {
-      if (role === ADMINISTRATOR) {
-        return <Fragment key={`${role}_${permission}`}>{hasPermission}</Fragment>;
-      }
-      if (role === MEMBER || role === CUSTOMER) {
-        if (PERMISSIONS_MAP[role][permission]) {
-          if (PERMISSIONS_MAP[role][permission] === ALL) {
-            return (
-              <Fragment key={`${role}_${permission}`}>
-                {hasPermission}
-                {hasPermission}
-              </Fragment>
-            );
-          }
-          return (
-            <Fragment key={`${role}_${permission}`}>
-              {hasPermission}
-              {noPermission}
-            </Fragment>
-          );
-        }
-        return (
-          <Fragment key={`${role}_${permission}`}>
-            {noPermission}
-            {noPermission}
-          </Fragment>
-        );
-      }
+      const permissionsArray = [];
+
       if (PERMISSIONS_MAP[role][permission]) {
-        return <Fragment key={`${role}_${permission}`}>{hasPermission}</Fragment>;
+        permissionsArray.push(ownerPermission);
+        if (PERMISSIONS_MAP[role][permission] === ALL) {
+          permissionsArray.push(notOwnerPermission);
+        }
       }
-      return <Fragment key={`${role}_${permission}`}>{noPermission}</Fragment>;
+      return (
+        <td className={cx('permission-cell')} key={`${role}_${permission}`}>
+          <div className={cx('cell-content')}>
+            {permissionsArray.map((item, index) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <Fragment key={index}>{item}</Fragment>
+            ))}
```

Comment:
You don't need create `permissionsArray` to store permissions to render them using `map`. You can just 
```
{ PERMISSIONS_MAP[role][permission] && ownerPermission }
{ PERMISSIONS_MAP[role][permission] === ALL && notOwnerPermission }
```

Reply:
Fixed